ggml.c

```diff
@@ -1912,9 +1912,12 @@ struct ggml_numa_node {
 };
 
 struct ggml_numa_nodes {
+    uint32_t numa_strategy;
```

Comment:
`numa_strategy` type should be `enum ggml_numa_strategies`.

Reply:
This has now been fixed and committed